tests/unit/Makefile

```diff
@@ -60,15 +60,37 @@ $(VALGRIND_TESTS)::
 	@DYLD_LIBRARY_PATH="$(LIBCRYPTO_ROOT)/lib:$$DYLD_LIBRARY_PATH" \
 	LD_LIBRARY_PATH="$(LIBCRYPTO_ROOT)/lib:$$LD_LIBRARY_PATH" \
 	S2N_VALGRIND=1 \
-	valgrind --leak-check=full --run-libc-freeres=no -q --error-exitcode=9 --gen-suppressions=all --log-fd=2 --num-callers=40 --leak-resolution=high --undef-value-errors=no --trace-children=yes --suppressions=valgrind.suppressions \
+	valgrind \
+		--leak-check=full \
+		--leak-resolution=high \
+		--trace-children=yes \
+		--run-libc-freeres=no \
+		-q --error-exitcode=123 \
+		--error-limit=no \
+		--num-callers=40 \
+		--undef-value-errors=no \
+		--log-fd=2 \
+		--suppressions=valgrind.suppressions \
```

Comment:
You removed "--gen-suppressions=all". Was that intentional?

Reply:
Yes. `--gen-suppresions=all` will cause valgrind to output formatted "suppressions" of the type used in `valgrind.suppressions` to ignore the memory errors. This is entirely redundant information, since it is just a reformatted version of valgrind's normal output. Suppressions should be a matter of last resort, and not a regular activity so we don't need to be generating them in CI.

Reply:
And including it in PEDANTIC_VALGRIND_TESTS is also intentional? It seems a little odd to include the option for pedantic but not standard.

Reply:
Talked offline, including it in the pedantic valgrind tests was accidental.